cobalt/h5vcc/h5vcc_updater.cc

```diff
@@ -41,11 +41,15 @@ void H5vccUpdater::SetUpdaterChannel(const std::string& channel) {
     return;
   }
 
+  // Force an update if SetUpdaterChannel is called, even if
+  // the channel doesn't change, as long as it's not "prod" channel
   if (updater_module_->GetUpdaterChannel().compare(channel) != 0) {
     updater_module_->SetUpdaterChannel(channel);
-    updater_module_->CompareAndSwapChannelChanged(0, 1);
-    updater_module_->RunUpdateCheck();
+  } else if (channel.compare("prod") == 0) {
```

Comment:
I'm confused by `channel.compare("prod") == 0`, if `channel` is a std::string, why not just do  `channel == "prod"` ? The current idiom made me think `channel` was something special.